rapid7_insightvm_cloud/plugin.spec.yaml

```diff
@@ -0,0 +1,1336 @@
+      type: integer
+      required: false
+    tags:
+      description: Asset tags
+      title: Tags
+      type: '[]asset_tag'
+      required: false
+    total_vulnerabilities:
+      description: Total number of vulns
+      title: Total Vulnerabilities
+      type: integer
+      required: false
+    source:
+      description: Source of the asset
+      title: Source
+      type: string
+      required: false
+    unique_identifiers:
+      description: Uniqure aspects of the asset
+      title: Unique Identifiers
+      type: '[]identifiers'
+      required: false
+    new:
+      description: unknown
+      title: New
+      type: '[]string'
+      required: false
+    remediated:
+      description: unknown
+      title: Remediates
+      type: '[]string'
+      required: false
+
+  
+connection:
+  credentials:
+    title: API Key
+    description: API key from account
+    type: credential_secret_key
+    required: true
+    example: 9de5069c5afe602b2ea0a04b66beb2c0
+  region:
+    title: Region
+    description: the region in which the plugin will work
+    type: string
+    required: true
+    example: us
+
+actions:
+  start_scan:
+    title: Start Scan
+    description: Start an InsightVM scan of previously scanned devices
+    input:
+      asset_ids:
+        title: Asset IDs
+        description: IDs of the assets to scan
+        type: "[]string"
+        example: ["cdc978de-f683-4178-a1d9-d5a94a114b87-default-asset-135"]
+        required: false
+      name:
+        title: Name
+        description: The name of the scan
+        type: string
+        example: "test cloud scan"
+        required: true
+      ip:
+        title: IP
+        description: Primary IPv4 or IPv6 address of the asset
+        type: "[]string"
+        example: ["10.4.31.141"]
+        required: false
+      hostname:
+        title: Hostname
+        description: The hostname
+        type: "[]string"
+        example: ["fortigate-vm02.vuln.lax.rapid7.com"]
+        required: false
+    output:
+      id:
+        description: Identifier of the resource created
+        title: Scan ID
+        type: integer
+        example: ' {
+              "scans": [
+                {
+                  "asset_ids": [
+                    "cdc978de-f683-4178-a1d9-d5a94a114b87-default-asset-13"
+                  ],
+                  "engine_id": "c8970d2ee174180e202af83b66cd0c4b",
+                  "finished": null,
+                  "id": "7fc3d8ec-24a7-4835-84f2-8877fe96a29b",
+                  "name": "testing scan action"
+                }
+              ],
+              "unscanned_assets": []
+            }'
+        required: false
+  get_scan:
+    title: Get Scan
+    description: Get the status of a scan
+    input:
+      scan_id:
+        title: Scan ID
+        description: ID of the scan to obtain
+        type: string
+        example: "abb37782-df95-4cf6-b4c2-8d466ca57349"
+        required: true
+    output:
```

Comment:
Seems like there are other useful returned fields like `finsihed` and `started` on the response from get_scan. Can we include all those?